app/frontend/src/components/form/Location.vue

```diff
@@ -123,12 +123,16 @@
               Details (eg:
               <em>"1km from HWY 1 at 100 mile house north on Logging Road"</em>)
             </label>
-            <v-text-field v-model="tentDetails" dense flat outlined solo />
+            <v-text-field
+              v-model="tentDetails"
+              :rules="[v => v.length <= 255 || 'Please enter 255 characters or less']"
```

Comment:
are we actually limiting them to 255? is that ok?  because we can alter the table if we want to allow more...

Reply:
👍

Reply:
It'd be a relatively safe change to add a migration that expanded the text field's maximum size - it wouldn't be a data loss operation. Should definitely double check with requirements before implementing this restriction in.